index.js

```diff
@@ -73,6 +75,9 @@ module.exports.makeApp = function () {
     }
   })
 
+  // Users
+  userRoutes(app)
```

Comment:
Can't we define it as controller below? E.g.:

// Controllers
  app.use([
    dmsRoutes(),
    userRoutes()
])

Reply:
I think this one is a little different because we need to pass the `app` object in order to enable some middleware.

Reply:
ok, got it